docs/faq.md

```diff
@@ -177,3 +177,32 @@ There is no golden rule, but you can follow these recommendations:
 
 This warning only occurs if you use non-default settings in your vite config that can cause problems in combination with prebundleSvelteLibraries.
 You should not use prebundleSvelteLibraries during build or for ssr, disable one of the incompatible options to make that warning (and subsequent errors) go away.
+
+<!-- the following header generates an anchor that is used in logging, do not modify!-->
+
+### deprecated "svelte" field
+
+In the past, Svelte recommended using the custom "svelte" field in package.json to allow libraries to point at .svelte source files.
+This field requires a custom implementation to resolve, so you have to use a bundler plugin and this plugin needs to implement resolving.
+Since then, node has added support for [conditional exports](https://nodejs.org/api/packages.html#conditional-exports), which have more generic support in bundlers and node itself. So to increase the compatibility with the wider ecosystem and reduce the implementation needs for current and future bundler plugins, it is recommended that packages use the "svelte" exports condition.
+
+Example:
+
+```diff
+// package.json
+- "svelte": "src/index.js"
++ "exports": {
++   "./package.json": "./package.json",
++   "./*": {
++     "svelte": "./src/*",
```

Comment:
does this need to be this...

```suggestion
+     "svelte": "./src/*.svelte",
```

...in order for `library/src/Foo.svelte` to be importable as `library/Foo`? Or is that undesirable/ (Would that pattern even work?)

Reply:
ideally it would be `"./*.svelte":{"svelte":"./src/*.svelte"}` but that didn't work in my testing even though node docs show a similar pattern for `*.js` 

omitting the extension altogether might confuse tools that look for it and may cause problems when there is a foo.js and a Foo.svelte  (hello windows)

Reply:
I think Rich's suggestion is the correct way to support it. Using `"./*.svelte"` means they have to import as `my-lib/Foo.svelte`

Reply:
i'm not an expert on building sane exports map for a new custom condition, so definetely up for improvements. Unfortunately i couldn't make the case with .svelte in the condition work, would love for someone to show a working example.

Reply:
From https://github.com/nodejs/node/issues/46074#issuecomment-1370102411:

> As the creator of Rollup, I can empathise: allowing modules to import each other without specifying the .js extension was a silly mistake that added untold complexity for no good reason

If it's a bad idea for `.js` files, it's probably a bad idea for `.svelte` files too, right? I don't think I would change the example to make it work without the `.svelte` extension

Reply:
So what does an ideal svelte exports condition look like today? just exporting the index.js that lists all .svelte files? Or use a glob too so that deep importing components can work?

I'd love to have a guideline for package authors here. Prebundling has made it less needed to use deep imports but they can still be a valid choice.

Reply:
@dummdidumm @Rich-Harris  

ideally this faq entry contains an example of a svelte exports map that works for most/all library authors. Once we merge this PR and they start updating, many are going to have to do a breaking release. If we change the recommended setup again later that would be kind of  a d*ck move and put extra pressure on the whole ecosystem by introducing multiple major updates.

Any thoughts on how glob exports should be utilized if at all?

Reply:
I've never seen anyone use glob exports in the wild before. It's probably not a good practice. E.g. if you have a main component that uses a few sub-components, you probably don't want to export the sub-components. I think I would change the example to not use glob exports

In terms of `index.js` vs deep package path, I don't know if we're wanting to make a universal declaration on that yet. You could potentially show both examples and then link to https://github.com/sveltejs/vite-plugin-svelte/blob/main/docs/faq.md#what-is-going-on-with-vite-and-pre-bundling-dependencies for a deeper discussion of what method to choose

Reply:
Hadn't thought about internal subcomponents. But i do think we have to provide guidance on how to allow deep imports and/or index imports. 

svelte-package stopped generating deep exports into package.json so it would be the library authors task to remember to add new components to that. But maybe thats preferable because with glob exports everything matched by the glob is public and that makes it easier to accidentally do a breaking change without realizing.

Reply:
removed the glob example and linked to the other faq entry.